.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,57 @@
+name: Bug Report
+description: File a bug report
+title: ""
+labels: []
```

Comment:
Opening the file on github has some syntax checking :)
https://github.com/Azure/azure-iot-middleware-freertos/blob/ewertons/issueform/.github/ISSUE_TEMPLATE/bug_report.yml
![image](https://user-images.githubusercontent.com/54871851/219220556-ee28a3b0-6e82-455b-9778-6cf2fa361047.png)

Reply:
Fixed. Thanks!